components/forms/TextField.vue

```diff
@@ -24,13 +26,15 @@ const { value, errorMessage } = useField<string>(name, undefined, {
       :color="color"
       :size="size"
       :disabled="disabled"
+      :trailing-icon="''"
 
       :icon="icon ?? ''"
       :loading="loading"
+      :trailing="trailing ?? true"
     >
-      <template #trailing>
+      <!-- <template #trailing>
         <slot name="trailing" />
-      </template>
+      </template> -->
```

Comment:
?

Reply:
Był z tym taki problem, że jak to było to właśnie ikonka pokazywała się i po lewej i po prawej stronie

Reply:
a jak to uzywales?

bo teraz funkcjonalnosc wywalasz